### DISCUSSION TITLE
Feature request - implement Animation

### DISCUSSION THREAD

**Discussion by ReinerBforartists** [hlky/stable-diffusion-webui#353] [Feature suggestions]
Hey friends,

There is a hacky Python script around to allow animations being rendered out from Stable Diffusion. And yesterday i stumbled across this video, were the video creator explained the use of the animation features of Stable Diffusion with Google Collab.

https://gist.github.com/karpathy/00103b0037c5aaea32fe1da1af553355
https://www.youtube.com/watch?v=JAhjeLiDvHY

Animation would be super great to create music videos for example. And so i want to suggest to implement animation into stable-diffusion-webui too

Kind regards
Reiner

**Comment by ReinerBforartists** [comment]
Another example and technique: 
https://replicate.com/andreasjansson/stable-diffusion-animation
https://github.com/andreasjansson/cog-stable-diffusion

**Comment by razvanab** [comment]
Here is another one that could help. I really hope you can implement this.

https://github.com/nateraw/stable-diffusion-videos

**Comment by codedealer** [comment]
txt2video is currently under development on the Streamlit version of the UI (you can check it on `dev` branch but beware of the bugs)
